### ISSUE TITLE
PLUGIN_TAGS not used in any plugin

### ISSUE THREAD

**Issue by ptitjes** [quodlibet/quodlibet#2283]
I just happen to see that there is a wonderful mechanism to categorize plugins.

Is there a reason that it is not used by any plugin ?

**Comment by urielz** [comment]
Right now it's hard to navigate the plugin list (so many!), and it's bound to get worst with time. If this means that we could potentially show the available plugins sorted by categories then it would be an awesome addition...

**Comment by ptitjes** [comment]
Also, a plugin may have multiple tags. So we could have multiple sets of categories with different points of view.

I tested it today, it works very fine and the plugin window already handles them in the drop down alongside with "All, Enabled, Disabled"...

**Comment by ptitjes** [comment]
@lazka, @declension, do you think we could use this mechanism to further categorize plugins ?
I mean a categorization that is orthogonal to the plugin type (which is now well used in the plugin window thanks to Nick).

**Comment by declension** [comment]
@ptitjes yes probably. That code is really old but seemed to be for that kind of purpose. 

I found that mixing enabled and disabled into that felt a bit weird though, or at least, not being able to query on a union of tags (i.e. _disabled_ and _playlists_, or whatever)... hence #2218

**Comment by ptitjes** [comment]
Yeah, I agree. Enabled, Disabled is kinda some 'dynamic' status about plugins whereas the assigned TAGs are more static data about them.

I like what you did in #2218.

However, I think we should go further after that. For instance, I would like that:
- we merge the main preferences and plugins preferences (it does make sense for us developers but I'm quite sure it does not for users).
- we add back/forward navigation in that new unified dialog and use more links between preferences to help users know they can configure something further
- we have full text search of preferences

**Comment by urielz** [comment]
@ptitjes I like your suggestions. However, the way I see it plugins are a strong feature of QL. I fear that moving them to the "Preferences" Menu might hide them a bit too much. Having said that I always thought that they were a bit out of place in the "File" menu. In my opinion they should be in the "Quodlibet" Menu right under "Preferences"...

**Comment by ptitjes** [comment]
@urielz I don't think that integrating them better will diminish their attractiveness. I'd say it is rather the opposite.
